### PR TITLE
Git and help icon buttons

### PR DESCRIPTION
**Pull request by VanessaScherma**
Git and help icons added on MenuToolbar.

### CODE REVIEW

**Comment by prasadtalasila** [comment]
@VanessaScherma please revert tge changes done to: package.json and .pre-commit.yaml files.

File: client/yarn.lock
Comment (prasadtalasila) [inline client/yarn.lock:1]: please revert changes to this file.

File: client/src/components/LinkButtons.tsx [hunk 1]
```diff
@@ -17,58 +17,69 @@ const ButtonRow = styled('div')({
...
   justifyContent: 'center',
   flexWrap: 'wrap',
-  gap: '50px',
 });
 
 interface IconButtonData {
   link: string;
   icon: React.ReactElement;
-  name: string;
+  name: string | undefined;
```
Comment (prasadtalasila) [inline client/src/components/LinkButtons.tsx:25]: Why do we need `undefined` here?

**Review by prasadtalasila — COMMENTED** [review]
@VanessaScherma, thanks for the updates. Please check the comments.

File: client/src/components/LinkButtons.tsx (hunk 1 above)
Comment (VanessaScherma) [reply client/src/components/LinkButtons.tsx:25]: This is needed since we decided to leave the NO_ICON (in LinkIconsLib.tsx) with the undefined name.

Comment (prasadtalasila) [reply client/src/components/LinkButtons.tsx:25]: We talked about using `topmenu` as names / prefix for the icons. Please use `code-github` and `help` as icon names.

**Pull request merged**